source/19-Psalms.usfm.db

```diff
@@ -5981,7 +5981,7 @@
 \q
 \v 5 What you have ordained is most sure;
 \q2 most sure shall your house stand inviolate,
-\q2 O \nd Lord\nd*, for ever and ever.
+\q2 O \nd Lord\nd*, [us:forever|cth:for ever] and ever.
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".